vsphere/internal/helper/storagepod/storage_pod_helper.go

```diff
@@ -518,3 +519,56 @@ func IsMember(pod *object.StoragePod, ds *object.Datastore) (bool, error) {
 	}
 	return true, nil
 }
+
+// get_recommended_datastore from storagePod
+func Get_recommended_Datastore(client *govmomi.Client,
```

Comment:
I believe that this function should be `GetRecommendDatastore`

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.

Reply:
Probably needs to be considered for use by `ExpandVirtualMachineCloneSpec` to help address the issue seen in https://github.com/hashicorp/terraform-provider-vsphere/issues/1323.

**Ryan Johnson**
Staff II Solutions Architect | VMware, Inc.